### PR TITLE
Import of ARC3/ARC4 system docs

### PR DESCRIPTION
**Pull request by jhodrien**
I've made an import of the ARC3/ARC4 pages, so we've got some content.

The table doesn't really carry across well, so I've flattened it.

### CODE REVIEW

File: book/systems/start.md [hunk 1]
```diff
@@ -1 +1,6 @@
-# Systems
\ No newline at end of file
+# Systems
+
+We provide and support a number of HPC clustersls
```
Comment (Sparrow0hawk) [inline book/systems/start.md:3]: Tiny typo here at the end

File: book/systems/arc4.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+2 nodes with 40 cores and 768GB of memory each and a hard disk drive within the node with 800GB of storage. This provides 80 cores of **high memory compute**.
+
+These can be requested by adding this line to submission scripts:
+
+`#$ -l node_type=40core-768G`
+
+#### GPGPU nodes
+3 nodes each with 40 cores, 192GB of system memory, a hard disk drive within the node with 800GB of storage and **4 x NVIDIA V100 GPUs**.
+See GPU section below on how to access these nodes
```
Comment (Sparrow0hawk) [inline book/systems/arc4.md:36]: Could add an in-page link here to below `(See GPU section below)[#using-the-v100-gpu-nodes]`

File: book/systems/start.md (hunk 1 above)
Comment (jhodrien) [reply book/systems/start.md]: Fixed in f1815eab

File: book/systems/arc4.md (hunk 2 above)
Comment (jhodrien) [reply book/systems/arc4.md]: Fixed in 8918764a

**Pull request merged**